.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ default config for macOS: &macos_defaults
   <<: *defaults
   resource_class: 'medium'
   macos:
-    xcode: '10.1.0'
+    xcode: '11.7'
```

Comment:
Not bumping to 12 here due to this issue: https://github.com/google/EarlGrey/issues/1447.